tfhe/src/integer/block_decomposition.rs

```diff
@@ -65,7 +65,7 @@ impl<const N: usize> RecomposableFrom<u8> for StaticUnsignedBigInt<N> {}
 impl<const N: usize> DecomposableInto<u64> for StaticUnsignedBigInt<N> {}
 impl<const N: usize> DecomposableInto<u8> for StaticUnsignedBigInt<N> {}
 
-#[derive(Copy, Clone)]
+#[derive(Clone)]
```

Comment:
why is Copy removed here we could want the Copy derive if T is copy right ?

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#/copy_iterator